packages/design-system/src/components/Drawer/Primitive/PrimitiveDrawer.scss

```diff
@@ -0,0 +1,34 @@
+@use '~@talend/design-tokens/lib/tokens';
+
+.primitive-drawer {
+	width: tokens.$coral-sizing-maximal;
+	height: 100%;
+	background: tokens.$coral-color-neutral-background;
+	display: flex;
+
+	&__header {
+		flex-grow: 0;
+		flex-shrink: 0;
+		flex-basis: tokens.$coral-sizing-l;
+		font: tokens.$coral-heading-m;
+
+		display: flex;
+	}
+
+	&__body {
+		flex-grow: 1;
+		flex-shrink: 1;
+		flex-basis: 0;
+		overflow: auto;
+
+		display: flex;
+	}
+
+	&__footer {
+		flex-grow: 0;
+		flex-shrink: 0;
+		flex-basis: 5.5rem;
```

Comment:
But not there?

Reply:
UX opinion: the buttons' height is 36px, and having a 40px footer would be a little tight

Reply:
Could be 40 + padding? Buttons would automatically make that zone taller